SceneRefAttributeValidator.cs

```diff
@@ -479,9 +572,11 @@ private static void ValidateRefLocationAnywhere(RefLoc loc, Component c, FieldIn
         private static bool IsEmptyOrNull(object obj, bool isArray)
         {
             if (obj is ISerializableRef ser)
+            {
                 return !ser.HasSerializedObject;
+            }
 
-            return obj?.Equals(null) != false || (isArray && ((Array)obj).Length == 0);
+            return obj == null || obj.Equals(null) || (isArray && ((IEnumerable)obj).CountEnumerable() == 0);
```

Comment:
Looks like you still need the extension for CountEnumerable

Reply:
I feel like ChatGPT today, Oh I'm sorry, I have made a mistake, here is the working code 🤣